executor/cook/config.py

```diff
@@ -35,6 +35,7 @@ def __init__(self,
                  flush_interval_secs=2,
                  max_bytes_read_per_line=1024,
                  max_message_length=512,
+                 memory_usage_interval_secs=15,
```

Comment:
I think 15 seconds is a pretty low default value, what about something like 5 minutes?

Reply:
The default is set to `3600` in https://github.com/twosigma/Cook/pull/618/files#diff-6a568f1963475d5ea436f8b8c51555caR88

Reply:
@pschorf happy with having the default there?